src/game_battler.cpp

```diff
@@ -261,21 +261,22 @@ bool Game_Battler::UseItem(int item_id) {
 		case RPG::Item::Type_armor:
 		case RPG::Item::Type_helmet:
 		case RPG::Item::Type_accessory:
-			return item->use_skill && UseSkill(item->skill_id);
+			return item->use_skill && UseSkill(item->skill_id, Main_Data::game_party->GetHighestLeveledActor());
 		case RPG::Item::Type_special:
-			return UseSkill(item->skill_id);
+			return UseSkill(item->skill_id, Main_Data::game_party->GetHighestLeveledActor());
```

Comment:
The observation that using of items that invoke skills uses the HighestLeveledActor instead of the current Actor is wrong.

Proof:
1. Create a "Special" Item that Invokes the Skill "Heal".
2. Set "Heal" to Fire element
3. Configure Fire as: A 999%, C 0%
4. Give the 1st actor Fire A and the 2nd actor Fire C
5. Try to use the item ingame. Result: 2nd actor can't use the item on itself.